src/pages/grid-test.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+import Layout from '../components/layout/layout';
+import SEO from '../components/seo';
+import { Grid, GridItem } from '../components/grid/grid';
+import styled from 'styled-components';
+
+const Card = styled.div`
+  color: black;
+  background-color: white;
+  border: 1px solid black;
+  text-align: center;
+`;
+
+const IndexPage: React.FC = () => (
+  <Layout>
+    <SEO title="Grid Example" />
+    <Grid>
+      <GridItem>
+        <h2>Grid</h2>
+      </GridItem>
+    </Grid>
+    <Grid>
+      <GridItem sm={6}>
+        <Card>xs=12 | sm=6</Card>
+      </GridItem>
+      <GridItem sm={6}>
+        <Card>xs=12 | sm=6</Card>
+      </GridItem>
+      <GridItem sm={12} md={4}>
+        <Card>xs=default | sm=12 | md=3</Card>
```

Comment:
Not sure if this page will be kept in the project or was just meant for illustration purposes.. but the configuration if the Grid Item & the Card content doesn't always match.

If this page is kept for documentation purposes, please check values. Also check if the code could be slimmed down/cleaned up. Otherwise it's a lot of repeating code.

Reply:
This page will be thrown out as soon as we have a component documentation like storybook or docz. I added it to have a visual feedback if the grid is even working 🙈